http/src/main/java/com/redhat/lightblue/client/http/LightblueHttpClient.java

```diff
@@ -119,21 +127,30 @@ public LightblueResponse metadata(LightblueRequest lightblueRequest) {
     @Override
     public LightblueResponse data(LightblueRequest lightblueRequest) {
         LOGGER.debug("Calling data service with lightblueRequest: " + lightblueRequest.toString());
-        try {
-            return callService(new LightblueHttpDataRequest(lightblueRequest)
-                    .getRestRequest(configuration.getDataServiceURI()));
-        } catch (RuntimeException e) {
-            throw new LightblueHttpClientException("Error sending lightblue request: " + lightblueRequest.getBody(), e);
-        }
+        return callService(new LightblueHttpDataRequest(lightblueRequest)
+                .getRestRequest(configuration.getDataServiceURI()));
```

Comment:
Method data(LightblueRequest) no longer wraps runtime exception.  I can see why that, at a glance, would seem bad.  Just want to be sure this change doesn't have unexpected side effects.  the difference is the request JSON won't be part of the stacktrace anymore.  I have mixed feelings about this.  1) it would be easier to debug yet 2) it means possible sensitive data is logged on the client.